labelbox/schema/dataset.py

```diff
@@ -293,7 +294,10 @@ def create_data_rows(self, items) -> "Task":
         task._user = user
         return task
 
-    def _create_descriptor_file(self, items, max_attachments_per_data_row=None):
+    def _create_descriptor_file(self,
+                                items,
+                                max_attachments_per_data_row=None,
+                                is_upsert=False):
```

Comment:
I understand you wanted to keep it [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself), but the code is arguably more complex as a result.

Reply:
I will address this in the follow-up work.